crates/rpc/rpc-testing-util/Cargo.toml

```diff
@@ -22,5 +22,8 @@ futures.workspace = true
 jsonrpsee = { workspace = true, features = ["client", "async-client"] }
 serde_json.workspace = true
 
+
+# Assertions
+pretty_assertions = "1.4.0"
```

Comment:
use similar_asserts

Reply:
> use similar_asserts

oh really sorry, didn't saw that you said similar_asserts, i don't know how i didn't read this, sorry again

updated